hypha/apply/users/views.py

```diff
@@ -28,47 +21,6 @@
 User = get_user_model()
 
 
-class LoginView(SuccessURLAllowedHostsMixin, TwoFactorLoginView):
-    redirect_authenticated_user = False
-
-    @method_decorator(sensitive_post_parameters())
-    @method_decorator(csrf_protect)
-    @method_decorator(never_cache)
-    def dispatch(self, request, *args, **kwargs):
-        if self.redirect_authenticated_user and self.request.user.is_authenticated:
-            redirect_to = self.get_success_url()
-            if redirect_to == self.request.path:
-                raise ValueError(
-                    "Redirection loop for authenticated user detected. Check that "
-                    "your LOGIN_REDIRECT_URL doesn't point to a login page."
-                )
-            return HttpResponseRedirect(redirect_to)
-        return super().dispatch(request, *args, **kwargs)
-
-    def get_success_url(self):
-        url = self.get_redirect_url()
-        return url or resolve_url(settings.LOGIN_REDIRECT_URL)
-
-    def get_redirect_url(self):
-        """Return the user-originating redirect URL if it's safe."""
-        redirect_to = self.request.POST.get(
-            self.redirect_field_name,
-            self.request.GET.get(self.redirect_field_name, '')
-        )
-        url_is_safe = is_safe_url(
-            url=redirect_to,
-            allowed_hosts=self.get_success_url_allowed_hosts(),
-            require_https=self.request.is_secure(),
-        )
-        return redirect_to if url_is_safe else ''
-
-    def get_context_data(self, **kwargs):
-        return super().get_context_data(
-            next=self.request.GET.get('next', ''),
-            **kwargs,
-        )
-
-
```

Comment:
@Parbhat It looks to me that this has been built in to django-two-factor-auth now so it can be removed?

Any other followup changes I have missed.

Reply:
Yeah, we can now use LoginView from the package instead of the custom one implemented in #1232. As redirect URL is implemented in the package https://github.com/Bouke/django-two-factor-auth/commit/90df643fc2f3eda5551b308df9cc200c283002f1#diff-b4dd7622064f0338584dd3ca5426d009

Looks like we do not need more changes.